### ISSUE TITLE
Windows MingW build errors

### ISSUE THREAD

**Issue by digdugg** [Cr4sh/openreil#17]
Working on compiling this via mingw on windows 7.  Running into a few issues but wanted to open an issue to document them so far.  

After running ./autogen.sh I received a number of ld errors when trying to find the python2.7 library.  On windows this is referenced via '-lpython27' and not '-lpython2.7'.  Additionally the patch has to prepend this with a '-L/pythonpath/libs'.

I've modified files manually and still working through issues as they come up.

**Comment by Cr4sh** [comment]
Yes, MinGW is shit and not supported officially yet. In some future I'm planing to update readme with build instructions that necessary to reproduce my MinGW build environment that I'm using to build OpenREIL binaries for Windows.

**Comment by digdugg** [comment]
If I get it to compile on my MinGW before docs are built out I'll certainly send over a PR.  Thanks for writing an awesome project.
